crates/sui-core/src/storage.rs

```diff
@@ -150,10 +149,9 @@ impl ReadStore for RocksDbStore {
         {
             let contents = self
                 .checkpoint_store
-                .get_full_checkpoint_contents_by_sequence_number(seq_num)
-                .expect("db error");
+                .get_full_checkpoint_contents_by_sequence_number(seq_num)?;
```

Comment:
In the issue we saw, the logic needs to continue to gather checkpoint content from individual txns and effects?

Reply:
In state-sync, it would just try to re-fetch the full checkpoint content from the network.
In checkpoint executor, it would try to gather the individual txs and effects.